libraries/fc/test/static_variant/test_static_variant.cpp

```diff
@@ -1,234 +0,0 @@
-#define BOOST_TEST_MODULE static_variant
```

Comment:
see some new functions being added in static_variant.hpp, this test needs to test those new apis.